spec/routes/api/project/location/private_subnet_spec.rb

```diff
@@ -207,7 +199,7 @@
       it "not authorized" do
         delete "/api/project/#{project_wo_permissions.ubid}/location/#{ps_wo_permission.display_location}/private-subnet/#{ps_wo_permission.name}"
 
-        expect(last_response.status).to eq(403)
+        expect(last_response).to have_api_error(403, "Sorry, you don't have permission to continue with this request.")
       end
     end
```

Comment:
New tests were added to this file in https://github.com/ubicloud/ubicloud/commit/752b59bb5a81978cb738bec1fbe173f19d72af09. Feel free to ignore and I'll update the new tests when this PR is merged.

Reply:
Thanks for the heads up. I added them.